GitSearch/Application/RepoInfo/RepoInfoViewController.swift

```diff
@@ -10,51 +10,42 @@ import UIKit
 
 class RepoInfoViewController: UITableViewController {
     
-    private let cellId = "common"
+    private let cellDefault = "default"
```

Comment:
Let's make naming more clear what it is, because now name implies its a some kind of cell itself.
```suggestion
    private let defaultReuseId = "default"
```

Reply:
changed it in another file too and also rename `cellDescription` to `descriptionReuseId`